ambari-logsearch/ambari-logsearch-web/src/app/components/logs-container/logs-container.component.ts

```diff
@@ -130,27 +134,28 @@ export class LogsContainerComponent implements OnInit, OnDestroy {
       this.filtersForm.valueChanges
         .filter(() => !this.logsContainerService.filtersFormSyncInProgress.getValue()).subscribe(this.onFiltersFormChange)
     );
-
+    // change the active tab when the active tab id changed in the URL
     this.subscriptions.push(
       this.activatedRoute.params.map((params: {[key: string]: any}) => params && params.activeTab)
         .subscribe(this.onActiveTabParamChange)
     );
-
+    // sync to filters form when the query params changed (only when there is no other way sync)
     this.subscriptions.push(
       this.activatedRoute.queryParams.filter(() => !this.queryParamsSyncInProgress.getValue()).subscribe(this.onQueryParamsChange)
     );
+    // get the current query params and sync them if any
     this.activatedRoute.queryParams.first().subscribe((params) => {
       if (!Object.keys(params).length) {
         this.syncFiltersToQueryParams(this.filtersForm.value);
       }
     });
-
+    // when the active tab id changed in the store we sync the tab filters to the query params
     this.subscriptions.push(
       this.logsStateService.getParameter('activeTabId').skip(1).subscribe(this.onActiveTabSwitched)
     );
-
+    // set the position of the filter panel depending on the scroll height: so it is fixed when it would be out from the screen
     this.subscriptions.push(
-      Observable.fromEvent(window, 'scroll').throttleTime(10).subscribe(() => {
+      Observable.fromEvent(window, 'scroll').subscribe(() => {
```

Comment:
is there any reason to remove throttleTime?

Reply:
Approving since this doesn't seem to critically reduce performance, but it's still the potential narrow place